tests/BootstrapTrait.php

```diff
@@ -142,4 +142,22 @@ protected static function getTestLogger(): TestLogger {
         $logger = static::container()->get(TestLogger::class);
         return $logger;
     }
+
+    /**
+     * Configure the container for connecting to the database.
+     */
+    protected static function initializeDatabase() {
+        /** @var TestInstallModel $model */
+        $model = static::container()->get(TestInstallModel::class);
+
+        /** @var \Gdn_Database $database */
+        $database = static::container()->get(\Gdn_Database::class);
+
+        $database->init([
+            'Host' => $model->getDbHost(),
+            'Dbname' => $model->getDbName(),
+            'User' => $model->getDbUser(),
+            'Password' => $model->getDbPassword(),
+        ]);
+    }
```

Comment:
Allow us to do some tests that touch the database without installing the entire site. This has to be called explicitly because it's hard to automate it without having to fix site installs, but I would like to make it automatic in the future.